news/src/main/java/org/exoplatform/platform/qa/ui/news/pageobject/NewsComposerManagement.java

```diff
@@ -0,0 +1,220 @@
+import static com.codeborne.selenide.Configuration.timeout;
+import static com.codeborne.selenide.Selectors.byText;
+import static com.codeborne.selenide.Selenide.switchTo;
+import static org.exoplatform.platform.qa.ui.news.NewsLocator.*;
+import static org.exoplatform.platform.qa.ui.selenium.locator.ActivityStreamLocator.ELEMENT_INPUT_ACTIVITY;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+
+public class NewsComposerManagement {
+  public ManageAlert alert;
+
+  public Button button;
+
+  public ManageFileTestBase mftb;
+  NavigationToolbar navigationToolbar;
+
+  /**
+   * constructor
+   *
+   * @param testBase TestBase
+   */
+  public NewsComposerManagement(TestBase testBase) {
+    this.alert = new ManageAlert(testBase);
+    this.button = new Button(testBase);
+    this.mftb = new ManageFileTestBase(testBase);
+    navigationToolbar = new NavigationToolbar(testBase);
+  }
+
+  /**
+   * Open news composer
+   */
+  public void openNewsComposer() {
+    NEWS_COMPOSER_LINK.shouldBe(visible).click();
+    switchTo().window(1);
+  }
+
+  /**
+   * Open news composer
+   */
+  public void openNewsEditor() {
+    NEWS_EDITOR_LINK.shouldBe(visible).click();
+    switchTo().window(2);
+  }
+
+  /**
+   * Click on the Post button
+   */
+  public void postNews() {
+    NEWS_COMPOSER_POST_BUTTON.waitUntil(enabled, timeout).click();
+  }
+
+  /**
+   * Click on the Update button
+   */
+  public void updateNews() {
+    NEWS_EDITOR_UPDATE_BUTTON.waitUntil(enabled, timeout).click();
+  }
+  /**
+   * Click on the Update and post button
+   */
+  public void updateAndPostNews() {
+    NEWS_EDITOR_UPDATE_AND_POST_BUTTON.waitUntil(enabled, timeout).click();
+  }
+  /**
+   * Add text into news title text box
+   *
+   * @param text String
+   */
+  public void fillTitleField(String text) {
+    NEWS_COMPOSER_TITLE_INPUT.waitUntil(visible, timeout).setValue(text);
+  }
+
+  /**
+   * Add text into news summary text box
+   *
+   * @param text String
+   */
+  public void fillSummaryField(String text) {
+    NEWS_COMPOSER_SUMMARY_INPUT.waitUntil(visible, timeout).setValue(text);
+  }
+
+  /**
+   * Add text into news content text box
+   *
+   * @param text String
+   */
+  public void fillBodyField(String text) {
+    info("----Add text into into news content text box-----");
+    switchTo().frame(NEWS_COMPOSER_CONTENT_INPUT);
+    ELEMENT_INPUT_ACTIVITY.click();
+    ELEMENT_INPUT_ACTIVITY.sendKeys(text);
+    switchTo().defaultContent();
+  }
+
+  /**
+   * Add text into news title text box
+   */
+  public void uploadIllustration() {
+    NEWS_COMPOSER_ILLUSTRATION_INPUT.uploadFromClasspath("illustration.jpg");
+  }
+
+  /**
+   * Create a News
+   *
+   * @param title
+   * @param body
+   */
+  public void createNews(String title, String body) {
+    openNewsComposer();
+
+    fillTitleField(title);
+    NEWS_DRAFT_SAVED.waitUntil(Condition.visible, Configuration.timeout);
```

Comment:
Not sure this test is useful since we don't know if the draft has really been saved. We should instead work on others test cases where we check the content of the saved draft. WDYT ?